File: CSharp/ReplaceWithAlphabetPosition.cs
public static class Kata
{
    public static string AlphabetPosition(string text)
    {
        text = text.ToLower();

        string alphabet = "abcdefghijklmnopqrstuvwxyz";
            
        string resultString = "";

        for (int i = 0; i < text.Length; i++)
        {
            if (alphabet.Contains(text[i]))
            {
                resultString += alphabet.IndexOf(text[i]) + 1 + " ";
            } else
            {
                continue;
            }
        }

        return resultString.Trim();
    }
}

File: Python/replace_with_alphabet_position.py
def alphabet_position(text):
    text = text.lower()

    alphabet = "abcdefghijklmnopqrstuvwxyz"

    result = ""

    for char in text:
        if char in alphabet:
            result += str(alphabet.find(char) + 1)
            result += " "
        else:
            result += ""

    return result.strip()